fmt/fmt_test.go

```diff
@@ -1,12 +1,15 @@
+//go:build google3
```

Comment:
I'm guessing you'd need to modify the go_test in google3 to include this build tag.

Reply:
Thanks for trying to fix this!
I didn't find other references inside Google to such comments.
I did find references inside Google that simply remove the "runfiles" import and delete the "runfiles.Path" call, could you try it to see if it works for you?

Reply:
I arbitrarily picked google3. `integration` might be a more descriptive name.

Removing the runfiles import and runfiles.Path call will break because this tests need the compiled binary. If I remove the runfiles.Path call, I'm not sure what to pass to `exec.Cmd`.

One option is to use a makefile that creates the binary at a testdata path and reference it from this test. Not sure that would work with bazel because you still need the runfiles for the data attribute. I don't think there's an easy way to have this type of test compile in both google3 and for external users. That's why I went with the build_tag. I don't particularly care about the test or plan to run it, but I need the code to compile and using a build tag would exclude the file from compilation.

Reply:
I see, in this case I will address this problem by excluding this file from open source: https://github.com/protocolbuffers/txtpbfmt/pull/38. Please revert this part of your pull request and I will accept it.